LockboxXCUITests/LockboxXCUITests.swift

```diff
@@ -75,27 +81,106 @@ class LockboxXCUITests: BaseTestCase {
             let regexpUid = "(uid=[a-z0-9]{0,32}$?)"
             let regexCode = "(code=[a-z0-9]{0,32}$?)"
             if let rangeUid = responseString?.range(of:regexpUid, options: .regularExpression) {
-                uid = (responseString?.substring(with:rangeUid))!
+                uid = String(responseString![rangeUid])
             }
 
             if let rangeCode = responseString?.range(of:regexCode, options: .regularExpression) {
-                code = (responseString?.substring(with:rangeCode))!
+                code = String(responseString![rangeCode])
             }
 
-            let finalCodeIndex = code.index(code.endIndex, offsetBy: -32)
-            let codeNumber = code[finalCodeIndex...]
-            let finalUidIndex = uid.index(uid.endIndex, offsetBy: -32)
-            let uidNumber = uid[finalUidIndex...]
-            self.completeVerification(uid: String(uidNumber), code: String(codeNumber)) {
+            if (code != nil && uid != nil) {
+                let codeNumber = self.getPostValues(value: code)
+                let uidNumber = self.getPostValues(value: uid)
+
+                self.completeVerification(uid: String(uidNumber), code: String(codeNumber)) {
+                    done()
+                }
+            } else {
                 done()
             }
         }
         task.resume()
     }
 
-    func test1LoginWithSavedLogins() {
+    private func getPostValues(value: String) -> String {
+        // From the regExp it is necessary to get only the number to add it to a json and send in POST request
+        let finalNumberIndex = value.index(value.endIndex, offsetBy: -32);
+        let numberValue = value[finalNumberIndex...];
+        return String(numberValue)
+    }
+
+    private func disconnectAccount() {
+        navigator.nowAt(Screen.LockboxMainPage)
```

Comment:
`nowAt` is a smell. If you need to use it, then you should be adding extra states into the graph.

However, I'm not sure you need it in this case.